internal/scripting/client.go

```diff
@@ -218,26 +218,53 @@ func (c *Client) StreamExecute(
 	return r, xerrors.WithStackTrace(err)
 }
 
+func receiveStreamResult(
```

Comment:
the function doesn't receive result, it make function for receive result.

Better to rename to, for example to `newReceiveStreamResultFunc`